packages/jellyfish-testing/src/testingwrapper.ts

```diff
@@ -0,0 +1,34 @@
+import { RegTestFoundationKeys } from '@defichain/jellyfish-network'
+import { ContainerGroup, DeFiDContainer, MasterNodeRegTestContainer } from '@defichain/testcontainers'
+import { Testing, TestingGroup } from '.'
+
+type InitDeFiContainerFn = (index: number) => DeFiDContainer
+function defaultInitDeFiContainer (index: number): DeFiDContainer {
+  return new MasterNodeRegTestContainer(RegTestFoundationKeys[index])
+}
+export class TestingWrapper {
+  create (): Testing
+  create (n: 1): Testing
+  create (n: number): TestingGroup
+  create (n: 1, init: InitDeFiContainerFn): Testing
+  create (n: number, init: InitDeFiContainerFn): Testing | TestingGroup
```

Comment:
```suggestion
  create (n: number, init: InitDeFiContainerFn): TestingGroup
```
Apologies I missed this case, when a number other than `1` _(hopefully no one puts a 0)_ is provided return type would be `TestingGroup` only.

Reply:
thanks you are right. zero is tricky perhaps `0 | 1 `shall do the trick? it shall create at least 1 instance then

Reply:
that makes sense and allows for correct output for both values if someone were to add a `0`